.github/workflows/benchmarks.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           bins: cargo-codspeed
 
       - name: Build the benchmark target(s)
-        run: cargo codspeed build ${{ matrix.benchmarks }}
+        run: cargo codspeed build -p benchmarks ${{ matrix.benchmark }}
```

Comment:
Shouldn't it be `-p bench`, since the profile is named `bench`?

Reply:
It's the name of the package, the `bench` profile is used by default so we don't need to specify `--profile`.